api/cpp/cbindgen.rs

```diff
@@ -157,8 +160,9 @@ fn gen_corelib(
         "sixtyfps_color_darker",
         "sixtyfps_image_size",
         "sixtyfps_image_path",
-        "TimerMode", // included in generated_public.h
-        "IntSize",   // included in generated_public.h
+        "TimerMode",      // included in generated_public.h
+        "IntSize",        // included in generated_public.h
```

Comment:
Unrelated... but do we really have IntSize in our public api?  It should probably be `Size`  or maybe `Size<int>`

Reply:
ACK, filed #909